dm/master/server.go

```diff
@@ -1395,29 +1408,23 @@ func withHost(addr string) string {
 	return addr
 }
 
-func (s *Server) removeMetaData(ctx context.Context, cfg *config.TaskConfig) error {
-	toDB := *cfg.TargetDB
-	toDB.Adjust()
-	if len(toDB.Password) > 0 {
-		toDB.Password = utils.DecryptOrPlaintext(toDB.Password)
-	}
-
+func (s *Server) removeMetaData(ctx context.Context, taskName, metaSchema string, toDBCfg config.DBConfig) error {
```

Comment:
```suggestion
func (s *Server) removeMetaData(ctx context.Context, taskName, metaSchema string, toDBCfg *config.DBConfig) error {
```

Reply:
can reduce the anooying `*` when invoking.

Reply:
fixed in https://github.com/pingcap/dm/pull/1999/commits/779794675eaa37084ab15e1b70447bbfe4538257